diff -urN OCSNG_LINUX_SERVER_1.01.orig/Apache/logrotate.ocsinventory-NG OCSNG_LINUX_SERVER_1.01/Apache/logrotate.ocsinventory-NG
--- OCSNG_LINUX_SERVER_1.01.orig/Apache/logrotate.ocsinventory-NG	2006-04-15 11:38:05.000000000 +0200
+++ OCSNG_LINUX_SERVER_1.01/Apache/logrotate.ocsinventory-NG	2007-05-09 10:36:19.000000000 +0200
@@ -15,7 +15,8 @@
 #
 # Rotate OCS Inventory NG Communication server logs weekly, only if not empty
 # Save 7 days old logs under compressed mode
-PATH_TO_LOG_DIRECTORY/*.log {
+/var/log/ocs-inventory-ng/*.log {
+	create 660 root http
 	daily
 	rotate 7
 	compress
diff -urN OCSNG_LINUX_SERVER_1.01.orig/Apache/ocsinventory.conf OCSNG_LINUX_SERVER_1.01/Apache/ocsinventory.conf
--- OCSNG_LINUX_SERVER_1.01.orig/Apache/ocsinventory.conf	2007-02-20 09:00:23.000000000 +0100
+++ OCSNG_LINUX_SERVER_1.01/Apache/ocsinventory.conf	2007-05-09 10:37:09.000000000 +0200
@@ -13,16 +13,16 @@
 # Which version of mod_perl we are using
 # For mod_perl <= 1.999_21, replace VERSION_MP by 1
 # For mod_perl > 1.999_21, replace VERSION_MP by 2
-PerlSetEnv OCS_MODPERL_VERSION VERSION_MP
+PerlSetEnv OCS_MODPERL_VERSION 2
 
 # Where to write detailled logs
-PerlSetEnv OCS_LOGPATH "PATH_TO_LOG_DIRECTORY"
+PerlSetEnv OCS_LOGPATH "/var/log/ocs-inventory-ng"
 
 # Database options
 # Replace DATABASE_SERVER by hostname or ip of MySQL server, generally localhost
-PerlSetEnv OCS_DB_HOST DATABASE_SERVER
+PerlSetEnv OCS_DB_HOST localhost
 # Replace DATABASE_PORT by port where running MySQL server, generally 3306
-PerlSetEnv OCS_DB_PORT DATABASE_PORT
+PerlSetEnv OCS_DB_PORT 3306
 # Name of database
 PerlSetEnv OCS_DB_NAME ocsweb
 PerlSetEnv OCS_DB_LOCAL ocsweb
