--- OCSNG_UNIX_SERVER-2.0.5/etc/logrotate.d/ocsinventory-server~	2012-04-01 16:05:55.000000000 +0200
+++ OCSNG_UNIX_SERVER-2.0.5/etc/logrotate.d/ocsinventory-server	2013-02-22 18:53:25.250736649 +0100
@@ -1,6 +1,7 @@
 # Rotate OCS Inventory NG Communication server logs daily
 # Save 7 days old logs in compressed mode
-PATH_TO_LOG_DIRECTORY/*.log {
+/var/log/ocs-inventory-ng/*.log {
+	create 660 root http
 	daily
 	rotate 7
 	compress
--- OCSNG_UNIX_SERVER-2.0.5/etc/ocsinventory/ocsinventory-server.conf~	2012-04-01 16:05:55.000000000 +0200
+++ OCSNG_UNIX_SERVER-2.0.5/etc/ocsinventory/ocsinventory-server.conf	2013-02-22 18:57:27.845324225 +0100
@@ -15,13 +15,13 @@
   # Which version of mod_perl we are using
   # For mod_perl <= 1.999_21, replace VERSION_MP by 1
   # For mod_perl > 1.999_21, replace VERSION_MP by 2
-  PerlSetEnv OCS_MODPERL_VERSION VERSION_MP
+  PerlSetEnv OCS_MODPERL_VERSION 2
 
   # Master Database settings
   # Replace DATABASE_SERVER by hostname or ip of MySQL server for WRITE
-  PerlSetEnv OCS_DB_HOST DATABASE_SERVER
+  PerlSetEnv OCS_DB_HOST localhost
   # Replace DATABASE_PORT by port where running MySQL server, generally 3306
-  PerlSetEnv OCS_DB_PORT DATABASE_PORT
+  PerlSetEnv OCS_DB_PORT 3306
   # Name of database
   PerlSetEnv OCS_DB_NAME ocsweb
   PerlSetEnv OCS_DB_LOCAL ocsweb
@@ -44,7 +44,7 @@
   # PerlSetVar OCS_DB_SL_PWD ocs
   
   # Path to log directory (must be writeable)
-  PerlSetEnv OCS_OPT_LOGPATH "PATH_TO_LOG_DIRECTORY"
+  PerlSetEnv OCS_OPT_LOGPATH "/var/log/ocs-inventory-ng/"
   
   # If you need to specify a mysql socket that the client's built-in
   #PerlSetEnv OCS_OPT_DBI_MYSQL_SOCKET "path/to/mysql/unix/socket"
